### PR TITLE
Update the MDT_BA permissions needed

### PR DESCRIPTION
**Pull request by ojrb**
Hi @mypil I add a reference to the minimum permission needed for the account MDT_BA, as requested in the issue, this PR solved the issue #1490

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/deploy-windows-mdt/create-a-windows-10-reference-image.md [hunk 1]
```diff
@@ -47,8 +47,8 @@ With Windows 10, there is no hard requirement to create reference images; howev
 -   Use the following settings for the New Deployment Share Wizard:
 -   Deployment share path: E:\\MDTBuildLab
 -   Share name: MDTBuildLab$
--   Deployment share description: MDT Build Lab
--   &lt;default&gt;
+-   Deployment share description: MDT Build Lab &lt;default&gt;
+-   Ensure that the MDT Build account (MDT_BA), has at least read permissions at share level on the deployment share.
```
Comment (JohanFreelancer9) [inline windows/deployment/deploy-windows-mdt/create-a-windows-10-reference-image.md]:
```suggestion
-   Ensure that the MDT Build account (MDT_BA) has at least read permissions at share level on the deployment share.
```

**Comment by ojrb** [comment]
@mypil  ready the changes suggested by @JohanFreelancer9

**Comment by mypil** [comment]
@greg-lindsay - Can you please assist to check, approve and merge the content of this PR based on issue #1490?

Thank you.

**Comment by mypil** [comment]
@greg-lindsay - We would like to follow up on this.  Can you please assist to check, approve and merge the content of this PR based on issue #1490? Thank you.

cc: @LauraWi

**Comment by mypil** [comment]
@greg-lindsay - Do you have any updates on this?

**Comment by greg-lindsay** [comment]
Closing this PR due to conflicting files and unclear steps.

**Pull request closed without merging**